src/plugins/saved_search/server/services/saved_searches/get_saved_searches.ts

```diff
@@ -38,6 +38,7 @@ export const getSavedSearch = async (savedSearchId: string, deps: GetSavedSearch
     savedSearchId,
     savedSearch.attributes,
     undefined,
-    await deps.searchSourceStart.create(searchSourceValues)
+    await deps.searchSourceStart.create(searchSourceValues),
+    false
```

Comment:
```suggestion
    Boolean(savedSearch.managed)
```
Should we pass the returned value here instead of hardcoding it?

Reply:
Good catch!

Reply:
Done in https://github.com/elastic/kibana/pull/175256/commits/06f2952aaed1f353da2469534244be4c35697243